src/main/java/sirius/biz/codelists/LookupTableController.java

```diff
@@ -51,6 +51,26 @@ public void suggestFromLookupTable(WebContext webContext,
                                        String tableName,
                                        String display,
                                        String extendedDisplay) {
+        suggestFromLookupTable(webContext, tableName, display, extendedDisplay, false);
+    }
+
+    /**
+     * Responds with possible suggestions from the given {@link LookupTable} using {@link AutocompleteHelper}.
+     *
+     * @param webContext               the web requests calling the autocomplete service
+     * @param tableName                the name of the table for which suggestions should be gathered
+     * @param display                  the requested {@link sirius.biz.codelists.LookupValue.Display display mode} for
+     *                                 the field label
+     * @param extendedDisplay          the requested {@link sirius.biz.codelists.LookupValue.Display display mode} for
+     *                                 the completion label
+     * @param considerDeprecatedValues whether deprecated values should be suggested
+     */
+    @Routed("/system/lookuptable/autocomplete/:1/:2/:3/:4")
```

Comment:
**Optional:** Should this really be part of the URL or not rather be a parameter that is read from the context? This doesn't feel very RESTy and would be even less breaking that way